### PR TITLE
Fixed #24895 -- Fixed loading a pair of squashed migrations with a dependency.

### PR DESCRIPTION
**Pull request by carljm**

### CODE REVIEW

**Comment by carljm** [comment]
buildbot, add to whitelist.

File: tests/migrations/test_loader.py [hunk 1]
```diff
@@ -267,6 +267,32 @@ def test_loading_squashed_complex_multi_apps(self):
...
+
+        loader.build_graph()
+
+        plan = set(loader.graph.forwards_plan(('app1', '4_auto')))
+        plan = plan - loader.applied_migrations
+        expected_plan = set([
+            ('app1', '4_auto'),
+            ('app1', '3_auto'),
+            ('app2', '1_squashed_2'),
+        ])
```
Comment (charettes) [inline tests/migrations/test_loader.py]: You could use a set literal here.

Comment (carljm) [inline tests/migrations/test_loader.py]: True. I was just being consistent with neighboring similar tests, which don't.

Comment (charettes) [inline tests/migrations/test_loader.py]: Ahh didn't notice that. I just remembered a recent commit that converted many of those to literals.

**Comment by timgraham** [comment]
Looks good besides more blank lines in the test than I'd opt for and missing release notes if you plan to backport to 1.8.

**Comment by carljm** [comment]
Thanks for reviews! Removed a blank line in the test, converted the new test and the one right before it to use set literals (the only tests in this file that needed that conversion), and added release note for 1.8.3. Will merge when CI is green.

**Pull request merged**